app/javascript/javascript/buildGraphOptions.js

```diff
@@ -46,14 +46,14 @@ const buildRangeSelector = ({ buttons, selected }) => ({
     states: {
       hover: {
         cursor: "pointer",
-        fill: "#09a7f0",
+        fill: "#00b2ef",
```

Comment:
Have you considered extracting a JS file for the colors? Similarly to what we have now in CSS.

Reply:
good idea. not doing not though